src/accelerate/accelerator.py

```diff
@@ -2243,14 +2243,18 @@ def gather(self, tensor):
         """
         return gather(tensor)
 
-    def gather_for_metrics(self, input_data):
+    def gather_for_metrics(self, input_data, use_gather_oject=False):
```

Comment:
```suggestion
    def gather_for_metrics(self, input_data, gather_non_tensors=False):
```

Perhaps using `gather_non_tensors` instead :)

Reply:
I think this can be confusing to ask the user to set `gather_non_tensors=True` with tensors.

Reply:
Fair point :)